contracts/contracts/sbtc-token.clar

```diff
@@ -0,0 +1,146 @@
+(define-constant ERR_NOT_AUTH (err u4)) ;; `tx-sender` or `contract-caller` is not the protocol caller
+(define-constant ERR_NOT_OWNER (err u5)) ;; `tx-sender` or `contract-caller` tried to move a token it does not own.
```

Comment:
Sorry, this still isn't right when comparing to the SIP010 transfer function, it should be:

```suggestion
(define-constant ERR_NOT_AUTH (err u5)) ;; `tx-sender` or `contract-caller` is not the protocol caller
(define-constant ERR_NOT_OWNER (err u4)) ;; `tx-sender` or `contract-caller` tried to move a token it does not own.
```